frontend/src/modules/organizers/components/OrganizersTable.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+  Avatar,
+  Center,
+  Flex,
+  Icon,
+  Spinner,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { OrganizerDTO } from 'generated-api';
+import { useContext, useEffect, useRef, useState } from 'react';
+import { FaBuilding, FaUserAlt } from 'react-icons/fa';
+import { useMutation } from 'react-query';
+import { Modal } from '../../../shared/components/elements';
+import { Dialog } from '../../../shared/components/elements/Dialog/Dialog';
+import { TableActions } from '../../../shared/components/table/TableActions';
+import { useToast } from '../../../shared/hooks';
+import { ApiContext } from '../../../shared/providers/ApiProvider';
+import { OrganizerEditForm } from './OrganizerEditForm';
+
+export const OrganizersTable = () => {
+  const api = useContext(ApiContext);
+  const toast = useToast();
+
+  const [organizers, setOrganizers] = useState<OrganizerDTO[] | undefined>([]);
+  const organizerToEdit = useRef<OrganizerDTO | null>(null);
+  const organizerToDelete = useRef<OrganizerDTO | null>(null);
+
+  const {
+    onOpen: onEditModalOpen,
+    isOpen: isEditModalOpen,
+    onClose: onEditModalClose,
+  } = useDisclosure();
+
+  const {
+    isOpen: isDeleteDialogOpen,
+    onOpen: onDeleteDialogOpen,
+    onClose: onDeleteDialogClose,
+  } = useDisclosure();
+
+  const fetchOrganizers = useMutation(() => api.getAllOrganizers(), {
+    onSuccess: (data) => {
+      setOrganizers(() => data.data);
+    },
+  });
+
+  const deleteUser = useMutation(
+    (organizerDTO: OrganizerDTO) => api.deleteUser(organizerDTO.id as string),
```

Comment:
this should be `deleteOrganizer`. it's also a backend problem that calling the `deleteUser` endpoint works without providing a correct id 🤔